consensus/clique/src/main/java/tech/pegasys/pantheon/consensus/clique/CliqueBlockInterface.java

```diff
@@ -29,6 +30,7 @@
 
   public static final Address NO_VOTE_SUBJECT =
       Address.wrap(BytesValue.wrap(new byte[Address.SIZE]));
+  public static final Address NO_VOTE_ADDRESS = Address.fromHexString("0");
```

Comment:
is this not the same as the NO_VOTE_SUBJECT?

Reply:
it is actually, the name threw me originally. changed to use the existing constant